rbd/rbd_pacific.go

```diff
@@ -0,0 +1,79 @@
+// +build !octopus,!nautilus
+
+package rbd
+
+// #cgo LDFLAGS: -lrbd
+// /* force XSI-complaint strerror_r() */
+// #define _POSIX_C_SOURCE 200112L
+// #undef _GNU_SOURCE
+// #include <errno.h>
+// #include <stdlib.h>
+// #include <rados/librados.h>
+// #include <rbd/librbd.h>
+import "C"
+
+import (
+	"unsafe"
+)
+
+const (
+	RBD_ENCRYPTION_FORMAT_LUKS1 = 0
+	RBD_ENCRYPTION_FORMAT_LUKS2 = 1
+)
+
+const (
+	RBD_ENCRYPTION_ALGORITHM_AES128 = 0
+	RBD_ENCRYPTION_ALGORITHM_AES256 = 1
+)
+
+type EncryptionFormatOptsLUKS1 struct {
+	Alg            uint
+	Passphrase     string
+	PassphraseSize uint
+}
+
+type EncryptionFormatOptsLUKS2 struct {
```

Comment:
this type has the same contents as `EncryptionFormatOptsLUKS1`, can a single type be used?

Reply:
I am trying to follow the same semantics as the librbd api where two structures were created for LUKS1 and LUKS2 to facilitate changes in one format while maintaining compatibility with existing code that uses the other.